app/serializers/v1/ssj/assigned_steps_serializer.rb

```diff
@@ -11,15 +11,21 @@ def serializable_hash
   end
 
   belongs_to :assignee, record_type: :people, id_method_name: :external_identifier,
-    serializer: V1::PersonSerializer do |process|
-    process.assignee
+    serializer: V1::PersonSerializer do |step|
+    step.assignee
   end
 
+  belongs_to :process, serializer: V1::Workflow::ProcessSerializer, record_type: :workflow_instance_process,
+    id_method_name: :external_identifier do |step|
+      step.process
+  end
```

Comment:
not sure you need to specify record_type, I assume it'd ask the step's process relationship that.